src/instance.rs

```diff
@@ -164,6 +168,20 @@ wrappable_struct!(
 
 class!(RubyExportedFunctions);
 
+#[rustfmt::skip]
+methods!(
+    RubyInstance,
+    itself,
+
+    fn ruby_exported_functions_method_exists(symbol: Symbol, _include_private: Boolean) -> Boolean {
```

Comment:
Why declaring `include_private` if it's not used?

Reply:
https://docs.ruby-lang.org/en/2.6.0/Object.html#method-i-respond_to_missing-3F

Reply:
:+1: